bigbluebutton/greenlight
# Fix Super Admin Role select

**Problem:**
A Super Admin cannot see its Role in Profile because the roles are scoped by the site provider, and Super Admin is "bn"
![image](https://user-images.githubusercontent.com/43917914/224401315-0635154f-f3e0-4fcf-a814-d40b17d4bbf0.png)

**Solution:**
Hardcode the Role input to "Super Admin" and disable it.
![image](https://user-images.githubusercontent.com/43917914/224401543-e4daa237-8e21-4a7e-9ac3-4a556a4bd8ca.png)

I don't see a reason why a Super Admin would want to change its role.

Another solution would be to create a new endpoint to fetch the Roles if the user is a super admin.

**How To Test:**

- Create a Super Admin
- Go to Profile and verify that Role is set to Super Admin and that you can't change it
- Go to Manage Users, and try to edit another user's role
- Log into a regular admin user and make sure that your role is not set to Super Admin
